source/_integrations/monoprice.markdown

```diff
@@ -22,6 +22,11 @@ Select the serial port to which Monoprice amplifier is connected, and name the s
 
 Sources can also be later edited from the integration options (gear icon in the top right when selecting the integration). In order to remove an existing source, you'll need to replace it with a space (simply removing its name will still keep it in place). Note that editing sources will remove the snapshot you may have saved.
 
+## Zone Management
+
+Devices and entities are created for each of the possible 18 zones, and can be enabled, disabled, and renamed through regular Home Assisant methods.
+By default, the first 6 zones (11..16) are enabled, while the other 12 zones possible through extensions (21..26, and 31..36) are disabled.
+
```

Comment:
```suggestion
Devices and entities are created for each of the possible 18 zones, and can be enabled, disabled and renamed through regular Home Assistant methods.
By default, the first 6 zones (11..16) are enabled, while the other 12 zones, possible through extensions (21..26, and 31..36), are disabled.

```

Reply:
Thanks @frenck.
The first change is a matter of convention (Oxford comma), so I'll go with whatever the convention in the docs is.
However, the additional commas in the second paragraph make it less readable, I believe. "while the other 12 zones possible through extensions" is a single phrase, and the zone number also belong in it.
A 3rd alternative would be "while the other 12 zones (21..26 and 31..36), which are possible through extensions, are disabled", but I think it's more cumbersome.

Reply:
We don't do Oxford comma's, see: <https://developers.home-assistant.io/docs/documentation_standards#general-documentation>

As for the other one, that is optional. 🤷‍♂

Reply:
ok, I removed the Oxford comma, and rephrased the second sentence to something that works better, I believe.